Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -14,6 +14,7 @@ import PackageModel
 import SourceControl
 import TSCBasic
 import TSCUtility
+import Foundation
```

Comment:
Was this intentional?

Reply:
good catch

Reply:
I suspect I know how it got added too...